_includes/releases/cloud/2023-04-10.md

```diff
@@ -0,0 +1,25 @@
+## April 10, 2023
+
+<h3> General changes </h3>
+
+- Contract customers can now [create {{ site.data.products.dedicated }} advanced clusters](../cockroachcloud/create-your-cluster.html#step-1-start-the-cluster-creation-process), which have all the features of {{ site.data.products.dedicated }} standard clusters, plus security features needed for [PCI compliance](../cockroachcloud/pci-dss.html). To upgrade your organization to a contract, [contact us](https://cockroachlabs.com/contact-sales).
```

Comment:
```suggestion
- Contract customers can now [create {{ site.data.products.dedicated }} advanced clusters](../cockroachcloud/create-your-cluster.html#step-1-start-the-cluster-creation-process), which have all the features of {{ site.data.products.dedicated }} standard clusters, plus security features needed for [PCI DSS compliance](../cockroachcloud/pci-dss.html). To upgrade your organization to a contract, [contact us](https://cockroachlabs.com/contact-sales).
```
It's the whole thing. :)

Reply:
Always? There are a few other places I can track down in the docs (and in the UI I think) where it's just PCI

Reply:
"Payment Card Industry Data Security Standard" is what it stands for, so you can see that just "PCI" is under-specified. See https://listings.pcisecuritystandards.org/documents/PCI_DSS-QRG-v3_2_1.pdf. This is a different PCI than "Peripheral Component Interconnect", which is a hardware bus / standard.